data/arrow_test.go

```diff
@@ -183,6 +183,13 @@ func goldenDF() *data.Frame {
 			float64(maxEcma6Int),
 			math.MaxFloat64,
 		}),
+		data.NewField("float64_nans", nil, []float64{
+			math.Inf(-1),
+			math.NaN(),
+			0,
+			math.NaN(),
```

Comment:
any reason for having this twice?

Reply:
no good reason....  just following the pattern for other test files:
```
		data.NewField("bool_values", nil, []bool{
			true,
			false,
			true,
			true,
			false,
		}),
```